recipes-core/microcode/intel-microcode_20170707.bb

```diff
@@ -45,7 +48,7 @@ do_install() {
 	install -d ${D}${base_libdir}/firmware/intel-ucode/
 	install ${WORKDIR}/microcode_${PV}.bin ${D}${base_libdir}/firmware/intel-ucode/
 	cd ${D}${base_libdir}/firmware/intel-ucode/
-	ln -sf microcode_${PV}.bin microcode_intel.bin
+	ln -sf microcode_${PV}.bin microcode.bin
```

Comment:
Won't that change break everything that uses `microcode_intel.bin`? (grub, installer, build scripts...)